Botkit/Microsoft.Bot.Sample.Slack/Bots/EchoBot.cs

```diff
@@ -11,13 +11,29 @@
 
 namespace Microsoft.Bot.Sample.Slack.Bots
 {
+    /// <summary>
+    /// An EchoBot class for that extends from the ActivityHandler.
```

Comment:
there's some word missing or in excess in this sentence

Reply:
yes, it is confusing. I will change it. Thanks for the feedback!